packages/eas-cli/src/commands/env/update.ts

```diff
@@ -0,0 +1,209 @@
+          nonInteractive,
+          required: false,
+          initial: existingVariable.value,
+        });
+        if (!value || value.length === 0) {
+          value = '';
+        }
+      }
+
+      const variable = await EnvironmentVariableMutation.createForAppAsync(
+        graphqlClient,
+        {
+          name,
+          value,
+          environment,
+          visibility,
+          overwrite: true,
+        },
+        projectId
+      );
+      if (!variable) {
+        throw new Error(
+          `Could not update variable with name ${name} on project ${projectDisplayName}`
+        );
+      }
+
+      Log.withTick(
+        `Updated ${chalk.bold(name)} with value ${chalk.bold(value)} on project ${chalk.bold(
+          projectDisplayName
+        )}.`
+      );
+    } else if (scope === EnvironmentVariableScope.Shared) {
+      const sharedVariables = await EnvironmentVariablesQuery.sharedAsync(graphqlClient, projectId);
+
+      if (!name) {
+        name = await selectAsync(
+          'Select variable',
+          sharedVariables.map(variable => ({
+            title: variable.name,
+            value: variable.name,
+          }))
+        );
+      }
+
+      const existingVariable = sharedVariables.find(variable => variable.name === name);
+      if (!existingVariable) {
+        throw new Error(
+          'Variable with this name already exists on this account. Please use a different name.'
+        );
+      }
+
+      if (!value) {
+        value = await promptVariableValueAsync({
+          nonInteractive,
+          required: false,
+          initial: existingVariable.value,
+        });
+        if (!value || value.length === 0) {
+          value = '';
+        }
+      }
+
+      if (!environment && link) {
+        environment = await promptVariableEnvironmentAsync(nonInteractive);
+      }
+
+      const variable = await EnvironmentVariableMutation.createSharedVariableAsync(
+        graphqlClient,
+        {
+          name,
+          value,
+          visibility,
+          overwrite: true,
+        },
+        ownerAccount.id
+      );
+
+      if (!variable) {
+        throw new Error(
+          `Could not update variable with name ${name} on account ${ownerAccount.name}`
+        );
+      }
+
+      Log.withTick(
+        `Updated ${chalk.bold(name)} with value ${chalk.bold(value)} on account ${chalk.bold(
+          ownerAccount.name
+        )}.`
+      );
+
+      if (link && environment) {
+        Log.withTick(
+          `Linking shared variable ${chalk.bold(name)} to project ${chalk.bold(
+            projectDisplayName
+          )}.`
+        );
+        await EnvironmentVariableMutation.linkSharedEnvironmentVariableAsync(
+          graphqlClient,
+          variable.id,
+          projectId,
+          environment
+        );
+        Log.withTick(
+          `Linked shared variable ${chalk.bold(name)} to project ${chalk.bold(projectDisplayName)}.`
+        );
+      }
```

Comment:
It makes sense to make `link` a part of the create command, but I'm not sure if it's necessary for update